app/main.py

```diff
@@ -1,8 +1,30 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
```

Comment:
The `Person` class constructor should initialize the `wife` and `husband` attributes to `None` to prevent potential `AttributeError` when these attributes are accessed later. Consider adding `self.wife = None` and `self.husband = None` in the constructor.